test/test-applications/integrations/dependency-libs/Samples.NoMultiLoader.Deps/Samples.NoMultiLoader.Deps.csproj

```diff
@@ -0,0 +1,22 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks Condition="'$(OS)' == 'Windows_NT'">net45;net461;netstandard2.0;netcoreapp3.1</TargetFrameworks>
+    <TargetFrameworks Condition="'$(OS)' != 'Windows_NT'">netstandard2.0;netcoreapp3.1</TargetFrameworks>
+    <LangVersion>latest</LangVersion>
+  </PropertyGroup>
+  
+  <PropertyGroup>
+    <ApiVersion Condition="'$(ApiVersion)' == ''">4.7.0</ApiVersion>
+  </PropertyGroup>
+
+
+  <ItemGroup Condition="$(TargetFramework.StartsWith('net4'))">
+    <Reference Include="System.Data" />
+  </ItemGroup>
+
+  <ItemGroup Condition="!$(TargetFramework.StartsWith('net4'))">
+    <PackageReference Include="System.Data.SqlClient" Version="$(ApiVersion)" />
```

Comment:
Super small nit: If you know that this version of the NuGet package triggers the issue and you don't plan to test other versions, feel free to remove the `ApiVersion` property

Reply:
done in https://github.com/DataDog/dd-trace-dotnet/pull/1055/commits/2bc41ac1d3bdba5a1eb989335800a9d59e2f9e3c